plugins/connection/lxd.py

```diff
@@ -63,6 +63,7 @@ class Connection(ConnectionBase):
     def __init__(self, play_context, new_stdin, *args, **kwargs):
         super(Connection, self).__init__(play_context, new_stdin, *args, **kwargs)
 
+        self._host = self._play_context.remote_addr.split(".")[0]
```

Comment:
Please note that this is deprecated, you need to use `self.get_option('remote_addr')` to access the current `remote_addr`. Also you need to do this when it's needed, not in advance.

So it's probably best to create a helper function which does this translation and which is called in the places where you added `self._host`.